pymc3/distributions/multivariate.py

```diff
@@ -88,13 +85,11 @@ def _quaddist(self, value):
 
         delta = value - mu
 
-        if self._cov_type == 'cov':
-            # Use this when Theano#5908 is released.
-            # return MvNormalLogp()(self.cov, delta)
-            dist, logdet, ok = self._quaddist_cov(delta)
-        elif self._cov_type == 'tau':
+        if self._cov_type == 'tau':
             dist, logdet, ok = self._quaddist_tau(delta)
         else:
+            # Use this when Theano#5908 is released.
```

Comment:
Pretty sure theano#5908 is released. Is there way to incorporate this comment? cc @aseyboldt